pallets/marketplace/src/lib.rs

```diff
@@ -663,9 +663,9 @@ pub mod pallet {
         /// Marketplace Logo URI updated. \[marketplace id, logo URI\]
         MarketplaceLogoUriUpdated(MarketplaceId, URI),
         /// Account added to disallow list for a marketplace.  \[marketplace id, account\]
-        AccountBlacklistedFromMarketplace(MarketplaceId, T::AccountId),
+        AccountAddedToDisallowList(MarketplaceId, T::AccountId),
```

Comment:
Change also the existing events AccountAddedToMarketplace and AccountRemovedFromMarketplace so they have the same format (AccountAddedToAllowList and  AccountRemovedFromAllowList)